client/state/action-types.js

```diff
@@ -23,6 +23,9 @@ export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
 export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
 export const FETCH_WPORG_PLUGIN_DATA = 'FETCH_WPORG_PLUGIN_DATA';
 export const NEW_NOTICE = 'NEW_NOTICE';
+export const POST_REQUEST = 'POST_REQUEST';
+export const POST_REQUEST_SUCCESS = 'POST_REQUEST_SUCCESS';
+export const POST_REQUEST_FAILURE = 'POST_REQUEST_FAILURE';
```

Comment:
just less duplication. it's really easy to fat finger one side of the const construction and not notice.

MY_REALLY_LONG_KEY = "MY_REALY_LONG_KEY"

vs

keyMirror( { MY_REALLY_LONG_KEY: t } )